### PR TITLE
tests:  Move skopeo test to rpm-ostree/container-deps

### PR DESCRIPTION
**Pull request by cgwalters**
And verify that `ostree container encapsulate` exists.  This
would have caught https://issues.redhat.com/browse/OCPBUGS-553

### CODE REVIEW

File: tests/kola/rpm-ostree/container-deps [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+set -xeuo pipefail
+
+. $KOLA_EXT_DATA/commonlib.sh
+
+# Verify this command exists since it's a hard dependency of ostree's container bits.
+skopeo experimental-image-proxy --help
+
+# Also this one verifies the linkage between ostree and rpm-ostree via extensions
+h=$(ostree container encapsulate --help)
+[[ "$h" =~ --copymeta ]] || (echo "missing ostree container" 1>&2; exit 1)
```
Comment (dustymabe) [inline tests/kola/rpm-ostree/container-deps]: we usually use `fatal` when reporting a failure

**Review by dustymabe — DISMISSED** [review]
LGTM

**Comment by dustymabe** [comment]
CI failed with:

```
[2022-08-25T20:58:05.224Z] + cosa fetch --strict --with-cosa-overrides
[2022-08-25T20:58:05.481Z] Config commit: e13a3fb37ba8ef0515ab6e6b5859ec0def236499
[2022-08-25T20:58:05.481Z] Using manifest: /srv/fcos/src/config/manifest.yaml
[2022-08-25T20:58:05.481Z] ls: cannot access '/srv/fcos/tmp/override/*.repo': No such file or directory
[2022-08-25T20:58:05.481Z] ERROR: no yum repo files were found
[2022-08-25T20:58:05.481Z] error: failed to execute cmd-fetch: exit status 1
```

I assume some issue with COSA?

**Comment by cgwalters** [comment]
OK right, I just had to rebase :surfer: this PR to fix the downgrades.

**Comment by HuijingHei** [comment]
Just want to confirm about the usage shows `ostree-container container ...`, is this expected?
```
[root@cosa-devsh ~]# ostree container encapsulate -h
ostree-container-container-encapsulate 0.8.1
Wrap an ostree commit into a container

USAGE:
    ostree-container container encapsulate [OPTIONS] <rev> <imgref> --repo <repo>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
        --cmd <cmd>...              Corresponds to the Dockerfile `CMD` instruction
        --copymeta <copymeta>...    Propagate an OSTree commit metadata key to container label
    -l, --label <label>...          Additional labels for the container
        --repo <repo>               Path to the repository

ARGS:
    <rev>       The ostree ref or commit to export
    <imgref>    Image reference, e.g. registry:quay.io/exampleos/exampleos:latest

[root@cosa-devsh ~]# ostree-container container encapsulate -h
-bash: ostree-container: command not found

[root@cosa-devsh ~]# rpm -q ostree
ostree-2022.2-5.el8.x86_64
```

**Review by HuijingHei — APPROVED** [review]

**Comment by cgwalters** [comment]
> Just want to confirm about the usage shows ostree-container container ..., is this expected?

It's a bug, but just a cosmetic one.  I did start on https://github.com/ostreedev/ostree-rs-ext/pull/345 which is tangentially related to this.

**Pull request merged**